tests/partitions_test.py

```diff
@@ -69,7 +70,10 @@ def test_partition_majority(network, args):
     initial_view = None
     with network.partitioner.partition(partition):
         try:
-            network.wait_for_new_primary(primary)
+            network.wait_for_new_primary(
+                primary,
+                timeout_multiplier=3,
+            )
```

Comment:
This is another `wait_for_new_primary` that's _expected to fail_, so I've reduced the multiplier in line with other instances. This brings the test runtime down.